### DISCUSSION TITLE
Relative Links not working

### DISCUSSION THREAD

**Discussion by NashJames** [community/community#15832] [Mobile]
My Profile README makes use of an SVG located in another repository. It doesn't show on the iOS app. Instead, I get an empty box where it should be.

**Comment by gavinmn** [comment]
Hey @NashJames — thanks for the feedback! I've created an issue to track this on our end.

**Comment by gavinmn** [comment]
In the meantime, to have this image displayed in your README on Mobile, I'd recommend using a non relative URL.

**Comment by moishinetzer** [comment]
This issue is still occurring. Any updates?
